### ISSUE TITLE
How to run

### ISSUE THREAD

**Issue by Aritoke9** [pyzhyk/btc-tx-adjustment-tool#1]
Hello owner please give know me how run it main.js or other file js

**Comment by pyzhyk** [comment]
- Clone the repository with `git clone https://github.com/pyzhyk/btc-tx-adjustment-tool.git` or download it from [here](https://github.com/pyzhyk/btc-tx-adjustment-tool/archive/main.zip) and unzip.
- Open `index.html` file in any browser.

Or you could simply use the online version: https://pyzhyk.github.io/btc-tx-adjustment-tool

**Comment by Aritoke9** [comment]
Is this a tool to find other people's transactions that have not been confirmed and send them to our address

**Comment by pyzhyk** [comment]
No, the tool only generates new transactions with adjusted output address. You have to sign the transaction and broadcast it afterwards, which means that you must be the owner of the input address. Otherwise, the new transaction can neither be signed nor confirmed.

**Comment by wise27-hacker** [comment]
can you post a video link on how it works.....

**Comment by purpleIQ** [comment]
Oh shucks! One needs private key.
